src/main/java/com/marklogic/developer/corb/Manager.java

```diff
@@ -637,7 +637,8 @@ protected void setJobServer(JobServer jobServer) {
 
     private void stopJobServer() {
         if (jobServer != null) {
-            jobServer.stop(0);
+            // UI polls on interval of 5 seconds, so delay just a bit longer before shutting down
+            jobServer.stop(6000);
```

Comment:
We could add a log message informing the user of the wait to avoid confusion especially when monitoring quick jobs. Also, should we give the user an option either to wait or how long to wait etc.

Reply:
Does the code know if the web server is registered with job server and/or polling for job status? If it knows, then we can adjust the default wait accordingly.

Reply:
Not currently, it doesn't have anything to track if the UI is pinging, but the `jobServer` is only going to be not null if it was enabled. So regular jobs will skip over this chunk of code. It won't delay shutdown of a vanilla CoRB job, only if Job Server was enabled.